### PR TITLE
feat: add dotnet6 and WinUI

### PR DESCRIPTION
**Pull request by carlh98**
GitHub Issue: #

## Proposed Changes

 - Feature 

## What is the new behavior?

feat: add dotnet6 and WinUI

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [ ] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

File: src/library/ExtendedSplashScreen.Uno.WinUI/IExtendedSplashScreen.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Nventive.ExtendedSplashScreen.WinUI
```
Comment (jeanplevesque) [inline src/library/ExtendedSplashScreen.Uno.WinUI/IExtendedSplashScreen.cs]: Shouldn't the interface be the same for all platforms and targets?

File: src/library/ExtendedSplashScreen.Uno.WinUI/Implementations/ExtendedSplashScreen.Android.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
+﻿#if WINUI && __ANDROID__
```
Comment (jeanplevesque) [inline src/library/ExtendedSplashScreen.Uno.WinUI/Implementations/ExtendedSplashScreen.Android.cs]: Could code be shared instead of being duplicated?

**Comment by jeanplevesque** [comment]
@carlh98 I think something broke with the PR. There are almost no changes 🤔

**Comment by carlh98** [comment]
@jeanplevesque add to fix the build without dotnet6 first, I'm adding back the changes

**Comment by jeanplevesque** [comment]
> @jeanplevesque add to fix the build without dotnet6 first, I'm adding back the changes

I see, I encourage you to do 2 separate PRs. 😊

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**